src/js/containers/aboutTheData/AgenciesContainer.jsx

```diff
@@ -89,7 +90,12 @@ const AgenciesContainer = ({
                         row.populate({ ...d, federalTotal });
                         return row;
                     });
-                    dispatch(setTableData(activeTab, parsedResults));
+                    if (searchTerm) {
```

Comment:
It would probably be faster to just map results using stringReplace here than having a check every time a drilldown cell is rendered.